parquet/src/main/java/org/apache/iceberg/parquet/ParquetDictionaryRowGroupFilter.java

```diff
@@ -385,12 +379,7 @@ private static boolean hasNonDictionaryPages(ColumnChunkMetaData meta) {
       encodings.remove(Encoding.RLE);
       encodings.remove(Encoding.BIT_PACKED);
 
-      if (encodings.isEmpty()) {
-        return false; // no encodings other than dictionary or rep/def levels
-      }
-
-      return true;
-
+      return !encodings.isEmpty(); // no encodings other than dictionary or rep/def levels
```

Comment:
This comment applies to the case where it is empty, so this is a little confusing now that the cases are combined. Could you move it to the previous line and add "when empty, no encodings ..." to clarify?

Reply:
Good point, I've updated the comment.